common/display/display-en.html

```diff
@@ -0,0 +1,970 @@
+---
+{
+	"title": "Display",
+	"language": "en",
+	"altLangPage": "display-fr.html",
+	"description": "Bootstrap and WET utility classes showcasing elements display options such as float, visibility, clearing, opacity, and toggling, accompanied by practical examples to test and observe different content layouts",
+	"dateModified": "2023-08-01"
+}
+---
+
+<p>This page showcases Bootstrap display utility features including some integration from wet-boew like toggling content.</p>
+
+<h2>On this page:</h2>
+
+<ul>
+	<li><a href="#float">Float</a></li>
+	<li><a href="#clear">Clear</a></li>
+	<li><a href="#toggling-content">Toggling content</a></li>
+	<li><a href="#visibility">Visibility</a></li>
+	<li><a href="#opacity">Opacity</a></li>
+	<li><a href="#wet-toggling-content">WET - Toggling content</a></li>
+	<li><a href="#col-count">Col count</a></li>
+	<li><a href="#sizing-controls">Sizing controls</a></li>
+</ul>
+
+<h2 id="bootstrap">Bootstrap</h2>
+
+<h3 id="float">Float</h3>
+
+<h4>Bootstrap 3 <code> .clearfix</code>working example</h4>
+<div class="well">
+	<div class="row">
+		<div class="col-md-3 well" style="background-color: #ffffff; float: left;">Column 1</div>
+		<div class="col-md-3 well" style="background-color: #ffffff; float: left;">Column 2</div>
+		<div class="col-md-3 well" style="background-color: #ffffff; float: left;">Column 3</div>
+		<!-- Add clearfix class to clear floated elements -->
+		<div class="clearfix"></div>
+	</div>
+	<p class="well">This is some content below the floated columns. Without clearfix, this content would overlap with the floated columns. But clearfix ensures proper layout by clearing the floats.</p>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="row"&gt;
+	&lt;div class="col-md-3 well" style="background-color: #ffffff; float: left;"&gt;Column 1&lt;/div&gt;
+	&lt;div class="col-md-3 well" style="background-color: #ffffff; float: left;"&gt;Column 2&lt;/div&gt;
+	&lt;div class="col-md-3 well" style="background-color: #ffffff; float: left;"&gt;Column 3&lt;/div&gt;
+	&lt;!-- Add clearfix class to clear floated elements --&gt;
+	&lt;div class="<strong>clearfix</strong>"&gt;&lt;/div&gt;
+&lt;/div&gt;
+&lt;p class="well"&gt;This is some content below the floated columns. Without clearfix, this content would overlap with the floated columns. But clearfix ensures proper layout by clearing the floats.&lt;/p&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>Bootstrap 3 <code>.center-block</code> class working example</h4>
+<div class="well">
+	<div class="center-block well" style="width: 200px; height: 200px; background-color: #ffffff;">
+		This element is centered horizontally within its parent container.
+	</div>
```

Comment:
May be an example that use an image will be easier to demo "centre-block" without any specific width.